.github/workflows/docker_cd_gpu.yml

```
@@ -0,0 +1,24 @@
+name: Docker GPU
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  docker-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get the version
+        id: get_version
+        run: echo VERSION=${GITHUB_REF/refs\/tags\//} >> $GITHUB_OUTPUT
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - run: |
+         make DEVICE=gpu build push
+         make TAG=$VERSION DEVICE=gpu build push
```

I know that's not new, but it seems to be missing one space indent. It works anyway, but I think that's why GitHub struggles to format/color correctly.
```suggestion
           make DEVICE=gpu build push
           make TAG=$VERSION DEVICE=gpu build push
```